Formula/google-protobuf.rb

```diff
@@ -3,15 +3,9 @@
 class GoogleProtobuf < Formula
   homepage "https://github.com/google/protobuf/"
   head "https://github.com/google/protobuf.git"
-  url 'https://github.com/google/protobuf/releases/download/v2.6.1/protobuf-2.6.1.tar.bz2'
-  version "2.6.1"
-  sha1 '6421ee86d8fb4e39f21f56991daa892a3e8d314b'
-
-  devel do
-    url "https://github.com/google/protobuf/archive/v3.0.0-alpha-2.tar.gz"
-    sha256 "46df8649e2a0ce736e37f8f347f92b32a9b8b54d672bf60bd8f6f4d24d283390"
-    version "3.0.0-alpha-2"
-  end
```

Comment:
Do we want to eventually remove this file and depend on the formula in Homebrew's repo?